app/javascript/packs/application.js

```diff
@@ -3,11 +3,12 @@
 // a relevant structure within app/javascript and only use these pack files to reference
 // that code so it'll be compiled.
 
-require("@rails/ujs").start()
-require("turbolinks").start()
-require("@rails/activestorage").start()
-require("channels")
-
+require("@rails/ujs").start();
+require("turbolinks").start();
+require("@rails/activestorage").start();
+require("channels");
+import "bootstrap";
+import "/Users/justinschreiner/Desktop/BNB/library/app/assets/stylesheets/application.css";
```

Comment:
Ok, here's what I would recommend for consistency's sake. Since you've added bootstrap via yarn, make another styles.scss down here and grab all of your custom styles out of the `app/assets/stylesheets` folder and put them inside of styles.scss then import that here in your application.js. That way you won't be using 2 different pipelines for styles and it will be more consistent. Then you can drop the `stylesheet_link_tag "application"` from your `application.html.haml`